src/routes/admin/+page.svelte

```diff
@@ -12,8 +12,8 @@
   const navItems = [
     new NavItem("/admin/frontdesk", "Bestillingsdisk"),
     // new NavItem("/admin/kitchen", "Kjøkken"),
-    new NavItem("/display", "Visning")
-    // new NavItem("/admin/screenmessage", "Endre skjerm melding"),
+    new NavItem("/display", "Visning"),
+    new NavItem("/admin/screenmessage", "Endre skjerm melding")
```

Comment:
Kunne vi hatt en egen config-side, heller enn å fylle navbaren med lenker for hver instilling? Kan være i admin dashboard fra #48 heller enn navbaren

Reply:
Dette er admin dashboarden. Jeg har kalt dem NavItem her også, men det er muligens litt forrvirende. AdminPage er kanskje bedre?

Reply:
> Dette er admin dashboarden. Jeg har kalt dem NavItem her også, men det er muligens litt forrvirende. AdminPage er kanskje bedre?

Åja, jeg la ikke merke til det pga. navnet. Ja, det er mye bedre.